change/@fluentui-react-infobutton-d4b92f9d-bdf3-44d0-be75-074ba1facfba.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "chore: Renaming content slot to info and updating InfoLabel to use InfoButton's shorthand.",
```

Comment:
this change of API is  a breaking change correct ? thus we should reflect that in this message so consumers are aware ( breaks are expected in alpha/beta )